### PR TITLE
:bulb: Update setup readme

### PR DESCRIPTION
**Pull request by Yexan**
Hi,

Here is an update on setup readme to ensure the developer has pushed his firebase modifications before to start Circle CI configuration step.
It might sound obvious but if you forget that step and go directly to the next step, your Circle CI configuration will never work :)

Thanks for your project ! 😊🙏🎉

### CODE REVIEW

**Comment by Yexan** [comment]
@kefranabg Any update or I should close it ?

**Comment by tbetous** [comment]
Hi @Yexan !

Thanks for your PR. Do not close it, I'll check it this week as soon as possible.

**Comment by kefranabg** [comment]
@Yexan Sorry for that I've been busy these days 😉 Thanks @tbetous for taking this 👍 

Thanks for your work !

File: docs/setup/README.md [hunk 1]
```diff
@@ -53,6 +53,11 @@ npx firebase login
 # Select the project you've just created and use "default" as alias
 npx firebase use --add
 
+# Push your changes
+git add .firebaserc src/firebase/init.js
+git commit -m 'Add firebase config'
```
Comment (tbetous) [inline docs/setup/README.md]: It is ok for me ! Just change the commit message to be suitable with gitmoji. I have added a suggestion.

Comment (tbetous) [inline docs/setup/README.md]:
```suggestion
git commit -m ':wrench: Add firebase config'
```

**Review by tbetous — CHANGES_REQUESTED** [review]

**Review by tbetous — APPROVED** [review]

**Comment by tbetous** [comment]
@Yexan, thanks a lot for your contribution !

**Comment by Yexan** [comment]
You're welcome 😊

**Pull request merged**